### PR TITLE
create a rake file

### PR DESCRIPTION
**Pull request by VickTor61**

### CODE REVIEW

File: lib/tasks/hello.rake [hunk 1]
```diff
@@ -0,0 +1,3 @@
+task :hello do
+  puts 'hello'
```
Comment (EnomaDebby) [inline lib/tasks/hello.rake]: Is this needed? Please remove files not needed

**Review by EnomaDebby — CHANGES_REQUESTED** [review]

File: lib/tasks/role.rake [hunk 2]
```diff
@@ -1,4 +1,13 @@
-desc 'Say hello!'
-task :hello_world do
-  puts "Hello"
-end
\ No newline at end of file
+ task create_role: :environment do
```
Comment (EnomaDebby) [inline lib/tasks/role.rake:1]: Did you run this task locally?

File: lib/tasks/hello.rake (hunk 1 above)
Comment (VickTor61) [reply lib/tasks/hello.rake]: oh sorry it was for testing purposes

File: lib/tasks/role.rake (hunk 2 above)
Comment (VickTor61) [reply lib/tasks/role.rake:1]: yes ma and it created the date in the db

**Review by EnomaDebby — APPROVED** [review]

**Pull request merged**